rest-api-spec/src/main/resources/rest-api-spec/api/indices.get_settings.json

```diff
@@ -16,6 +16,10 @@
         }
       },
       "params": {
+        "master_timeout": {
+          "type": "time",
+          "description": "Specify timeout for connection to master"
+        },
```

Comment:
not needed, it's just unfortunate that due to the hierarchy of our classes the request exposes this.

Reply:
Per your earlier feedback, I added this throughout and this parameter now makes it through.  Should I remove?

Reply:
sorry about the confusion. I didn't mean that we should add support for it. This param doesn't do anything, even now that it's parsed does it? If that's case it makes little sense to add support to it on the REST layer. It is unfortunate that the java request exposes it. We could error in the high-level client when it gets set, just to warn users that it does nothing when set on this specific class and it makes no sense to set it.

Reply:
ehm.... I looked into this deeper, as it came up also in #29331 . I think that when parsed and passed to the transport action, the timeout actually does get applied. So you are right that this should be added to our SPEC and parsed in the REST layer! I thought that it had no effect as we are only reading the cluster state rather than writing it, but I was wrong.